src/components/proxy-display.js

```diff
@@ -6,12 +6,17 @@ import {
   Icon,
 } from '@folio/stripes/components';
 
-export default function ProxyDisplay({ proxy, proxyTypes, inheritedProxyId }) {
+const ProxyDisplay = ({
```

Comment:
`<ProxyDisplay>` was removed from this directory in the [PR](https://github.com/folio-org/ui-eholdings/pull/1386). Also tests was implemented for it there too. Please remove changes for `<ProxyDisplay>` from the current PR. You could add tests for it if you want, but it's looks like is all covered yet.